src/Lesson.java

```diff
@@ -0,0 +1,46 @@
+public class Lesson {
+    Long id;
+    String name;
+    Long homework;
```

Comment:
Normally, you would have an object `Homework homework` but not an id. An id will be stored in the database because you have a foreign key and this is a way how the relations are kept in DB. So, having this in mind, change `Long homework` to `Homework homework`